scripts/us_cdc/non_infectious_diseases/README.md

```diff
@@ -11,14 +11,6 @@ To download the data,
 3. Scroll down towards the footer of the page to find a link to download data in
    excel format. Select the link called *Download all NORS Dashboard data (Excel)*. This will download all the required dataset.
```

Comment:
The section with a link to https://data.cdc.gov/Foodborne-Waterborne-and-Related-Diseases/NORS/5xkq-dg7x/about_data from your previous commit was removed, please add it back.